libraries/provider_hab_service.rb

```diff
@@ -0,0 +1,69 @@
+# Copyright:: Copyright 2016, Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "chef/provider/service"
+
+class Chef
+  class Provider
+    class Service
+      class HabSup < Chef::Provider::Service
+        include HabitatCookbook::Helpers
+
+        use_inline_resources
+
+        provides :hab_service
+
+        action :start do
+          declare_resource(:systemd_unit, "#{service_name}.service") do
+            action :create
+            content lazy { unit_content }
```

Comment:
I missed this when I moved the `SSL_CERT_FILE` to the resource's property - that does `lazy{}` now.